toolchain/lex/lex.cpp

```diff
@@ -0,0 +1,17 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "toolchain/lex/lex.h"
+
+#include "toolchain/lex/lexer.h"
+
+namespace Carbon::Lex {
+
+auto Lex(SharedValueStores& value_stores, SourceBuffer& source,
+         DiagnosticConsumer& consumer) -> TokenizedBuffer {
+  Lexer lexer(value_stores, source, consumer);
+  return std::move(lexer).Lex();
```

Comment:
Consider combining these:
```suggestion
  return Lexer(value_stores, source, consumer).Lex();
```